sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -569,3 +569,27 @@ public abstract partial class ObjectSerializer
         public virtual System.Threading.Tasks.ValueTask<System.BinaryData> SerializeAsync(object? value, System.Type? inputType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
 }
+namespace Azure.Messaging
+{
+    public partial class CloudEvent
+    {
+        public CloudEvent(string source, string type, System.BinaryData data, string dataContentType, Azure.Messaging.CloudEventDataFormat dataFormat = Azure.Messaging.CloudEventDataFormat.Binary) { }
+        public CloudEvent(string source, string type, object jsonSerializableData, System.Type? dataSerializationType = null) { }
+        public System.BinaryData? Data { get { throw null; } set { } }
+        public string? DataContentType { get { throw null; } set { } }
+        public string? DataSchema { get { throw null; } set { } }
+        public System.Collections.Generic.IDictionary<string, object?> ExtensionAttributes { get { throw null; } }
+        public string Id { get { throw null; } set { } }
+        public string Source { get { throw null; } set { } }
+        public string? Subject { get { throw null; } set { } }
+        public System.DateTimeOffset? Time { get { throw null; } set { } }
+        public string Type { get { throw null; } set { } }
+        public static Azure.Messaging.CloudEvent? Parse(System.BinaryData jsonEvent, bool strict = true) { throw null; }
```

Comment:
Why doesn't this one return an array? but the next one does?

Reply:
This is meant as a convenience when working with the set of event subscribers that expose the message body as BinaryData (e.g. Service Bus, Event Hubs, Storage Queues). There would only ever be JSON for a single cloud event in the body.

Reply:
Typically we try to avoid overloads of the same method having different return types.

Reply:
Yes but the return types are the same type, it is just whether there is a single or multiple.

Reply:
Yeah, that's a different type. Why wouldn't I want to parse a string with a single event or BinaryData with multiple?

Reply:
Yes, I just mean they are dealing with the same type, either the type itself or an array of the type. I think this came up before and we decided if the type is consistent then it is probably okay to have the return type differ by single or multiple. 

This is certainly a tradeoff - in the common case with BinaryData you would have a single event, whereas with string you are more likely to have multiple events. If you try to use the BinaryData Parse with multiple events present an exception would be thrown.

Reply:
So why complicate things? 

Why not have `EventData Parse` and `EventData[] ParseEvents ` ?

Reply:
Well.. now it seems obvious.